app/views/transactions/_transaction.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr>
+  <td><%= transaction.created_at.strftime('%F') %></td>
+  <% if transaction.user != nil %>
+    <td><%= transaction.user.name %></td>
+  <% else %>
+    <td>Not Found</td>
+  <% end %>
```

Comment:
use safe navigation here.
And always move this kind of logic in helper methods.